README.md

```diff
@@ -0,0 +1,40 @@
+# Dewy LangChain Plugin
+
+This package provides Dewy integration for LangChain.
+
+Example:
+
+```
+from langchain_core.runnables import RunnableParallel, RunnablePassthrough
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import ChatPromptTemplate
+
+from dewy_langchain import Retriever
+
+retriever = Retriever(collection_id=collection_id)
+prompt = ChatPromptTemplate.from_messages(
+    [
+        (
+            "system",
+            """
+            You're a helpful AI assistant. Given a user question and some retrieved content, answer the user question.
+            If none of the articles answer the question, just say you don't know.
+
+            Here is the retrieved content:
+            {context}
+            """,
+        ),
+        ("human", "{question}"),
+    ]
+)
+
+def format_chunks(chunks):
```

Comment:
It occurs to me somewhat silly that we:

1. Take a Pydantic model of our chunks
2. Convert it to the LangChain Document pydantic model
3. Convert that to a string using metadata fields

I think this makes sense to support this way as a drop in replacement since it implements the Retriever API, but I could see value in showing a version where the ChatPromptTemplate operates directly on Dewy chunks, rather than LangChain documents.